### PR TITLE
packaging: various packaging fixes

### PR DESCRIPTION
**Pull request by klihub**
- fix previous shellcheck 'fix' breaking scripts/build/docker-build-image
  - fix broken fedora cross-builds
  - reduce initial in-container dnf/yum/zypper operation cost by using a minimal viable package set
  - retain `fallback.cfg.sample` sample config name when installing it to `/etc/cri-resmgr`
  - use `/etc/default` for default options on deb-based distros and `/etc/sysconfig` only on rpm-based ones
  - add the necessary collateral for Ubuntu-20.04 cross-builds
  - add generic cross-build targets `cross-packages`, `cross-rpm`, and `cross-deb`

### CODE REVIEW

**Comment by klihub** [comment]
@marquiz, @kad Updated to reduce initial package resolution load, follow debian-ish/fedora-ish location of defaults on their respective distros, keep installed sample config as sample in /etc/cri-resmgr. Added generic package cross-building targets (already documented in `RELEASE.md`).

File: dockerfiles/cross-build/Dockerfile.centos-8 [hunk 1]
```diff
@@ -7,11 +7,9 @@ ARG CREATE_USER="build"
 ARG USER_OPTIONS=""
 ENV PATH /go/bin:/usr/local/go/bin:$PATH
 
-RUN yum install -y /usr/bin/bash /usr/bin/git /usr/bin/make \
-    /usr/bin/grep /usr/bin/sed /usr/bin/tr /usr/bin/find \
-    /usr/bin/sort /usr/bin/uniq \
-    /usr/bin/wget /usr/bin/tar /usr/bin/bzip2 \
-    kernel-devel rpm-build clang gcc
+RUN yum install -y rpm-build \
```
Comment (kad) [inline dockerfiles/cross-build/Dockerfile.centos-8:10]: in centos 8 yum is obsoleted in favour of dnf.

File: dockerfiles/cross-build/Dockerfile.suse-15.1 [hunk 2]
```diff
@@ -7,12 +7,10 @@ ARG CREATE_USER="build"
 ARG USER_OPTIONS=""
 ENV PATH /go/bin:/usr/local/go/bin:$PATH
 
-RUN zypper install -y bash git make grep \
-    sed findutils coreutils \
-    tar bzip2 wget \
-    clang gcc \
-    kernel-devel \
-    rpm-build
+RUN zypper install -y rpm-build \
```
Comment (kad) [inline dockerfiles/cross-build/Dockerfile.suse-15.1:10]: 15.2 was released. we might want to copy that dockerfile as 15.2 as well.

**Review by kad — DISMISSED** [review]
smalls nits, but otherwise lgtm.

File: dockerfiles/cross-build/Dockerfile.centos-8 (hunk 1 above)
Comment (klihub) [reply dockerfiles/cross-build/Dockerfile.centos-8:10]: Ok, I can fix that one and push once more.

File: dockerfiles/cross-build/Dockerfile.suse-15.1 (hunk 2 above)
Comment (klihub) [reply dockerfiles/cross-build/Dockerfile.suse-15.1:10]: Ok, just let me check it can build with that.

**Review by kad — APPROVED** [review]
lgtm

**Pull request merged**